Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -39,9 +41,40 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def find_conflict_pr(cls, remote, branch):
-        for pr in remote.pull_requests.find(head=branch, opened=True):
-            return pr
+    def radar_for_id(cls, radar_id):
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, radar.Tracker):
+                return tracker().client.radar_for_id(radar_id, additional_fields=['sourceChanges'])
+
+    @classmethod
+    def find_conflict_pr(cls, remote, radar_id):
+        """
+        Because we don't know what the target branch was just given the radar id,
+        we need to search for PRs with branches that start with the integration prefix
+        and have the first and last sha.
+        """
+        radar_obj = cls.radar_for_id(radar_id)
```

Comment:
It doesn't seem like we actually need a radar here, right? We just need any issue that we can extract commits from.

Reply:
We can avoid using a new function and calling radar client directly by doing `radar_obj = Tracker.from_string(radar_id)`. This would also work if a bugzilla is passed in. If we want to support bugzilla we can just change the variable names.